### PR TITLE
added chill guide

### PR DESCRIPTION
**Pull request by FieldmanJR1**
Added chill guide as part of #18

### CODE REVIEW

File: src/validator-guides/how-to-chill.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# How to chill
```
Comment (xcthulhu) [inline src/validator-guides/how-to-chill.md:1]:
```suggestion
# How to Chill
```

File: src/validator-guides/how-to-chill.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+The Golden Gate validator may opt to chill whenever he wants to prevent participation in staking for the upcoming era but does not want to unstake funds (e.g. when there is maintenance planned)
+
+Validator can do that by calling the `chill` [extrinsic ](../#extrinsic)in the [staking pallet](https://paritytech.github.io/substrate/master/pallet\_staking/pallet/enum.Call.html#variant.chill) following the steps:
+
+1.  Go to Developer-->Extrinsics
+
+    <figure><img src="../.gitbook/assets/image (4) (1).png" alt=""><figcaption></figcaption></figure>
```
Comment (xcthulhu) [inline src/validator-guides/how-to-chill.md:9]: There's no such asset.  Probably should link to an image in `src/`

**Comment by xcthulhu** [comment]
Images are not rendering, see the preview here: https://github.com/GoldenGateGGX/documentation/blob/aad3f1369e8635f238f7e9227f048975b0f5ca0b/src/validator-guides/how-to-chill.md

File: src/validator-guides/how-to-chill.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
+# How to chill
+
+The Golden Gate validator may opt to chill whenever he wants to prevent participation in staking for the upcoming era but does not want to unstake funds (e.g. when there is maintenance planned)
+
+Validator can do that by calling the `chill` [extrinsic ](../#extrinsic)in the [staking pallet](https://paritytech.github.io/substrate/master/pallet\_staking/pallet/enum.Call.html#variant.chill) following the steps:
+
+1.  Go to Developer-->Extrinsics
```
Comment (xcthulhu) [inline src/validator-guides/how-to-chill.md:7]:
```suggestion
1.  Go to Developer → Extrinsics
```

**Comment by uchu** [comment]
Cleanup. Closing open PRs here as was discussed. Maybe, we can re-use some parts of that in the upcoming documentation.

**Pull request closed without merging**